### ISSUE TITLE
Run artillery script on AWS Lambda 

### ISSUE THREAD

**Issue by gabor386** [artilleryio/artillery#1816]
Version info: 
```
artillery 2.0.0.30
node 18.13.0
npm 9.4.0
```

Running this command: 
```
artillery  run stage.yml --platform aws:lambda


```

I expected to see this happen:

-To run on AWS Lambda

Instead, this happened:

-Failed do to module issue (dependency) @oclif
i did reproduce on another windows machine,
Installed Node, aws cli, artillery and run some basic script. Maybe i am missing something 

the error code :
C:\Users\Desktop>artillery run stage.yml --platform aws:lambda
NOTE: AWS Lambda support is experimental. Not all Artillery features work yet.
For details please see https://docs.art/aws-lambda

λ Creating AWS Lambda function...

Bundling test data
stage.yml
Installing dependencies
undefined undefined null Error: spawnSync npm ENOENT
at Object.spawnSync (node:internal/child_process:1110:20)
at spawnSync (node:child_process:871:24)
at PlatformLambda.init (C:...\Roaming\npm\node_modules\artillery\lib\platform\aws-lambda\index.js:152:47)
at async Launcher.run (C:...\Roaming\npm\node_modules\artillery\lib\launch-platform.js:377:25) {
errno: -4058,
code: 'ENOENT',
syscall: 'spawnSync npm',
path: 'npm',
spawnargs: [ 'install' ]
}
Creating zip package
Preparing AWS environment...
Lambda function: artilleryio-49a32904-5417-4f6f-a1e9-a824150b311b
Region: us-east-1
AWS account: **********
[37d94b44-c016-4ea7-******]: A Lambda function has exited with an error. Reason: ArtilleryError
{
stdout: '',
stderr: 'node:internal/modules/cjs/loader:936\n' +
' throw err;\n' +
' ^\n' +
'\n' +
"Error: Cannot find module '@oclif/command'\n" +
'Require stack:\n' +
'- /var/task/node_modules/artillery/bin/run\n' +
' at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)\n' +
' at Function.Module._load (node:internal/modules/cjs/loader:778:27)\n' +
' at Module.require (node:internal/modules/cjs/loader:1005:19)\n' +
' at require (node:internal/modules/cjs/helpers:102:18)\n' +
' at Object. (/var/task/node_modules/artillery/bin/run:7:18)\n' +
' at Module._compile (node:internal/modules/cjs/loader:1105:14)\n' +
' at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)\n' +
' at Module.load (node:internal/modules/cjs/loader:981:32)\n' +
' at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n' +
' at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12) {\n' +
" code: 'MODULE_NOT_FOUND',\n" +
" requireStack: [ '/var/task/node_modules/artillery/bin/run' ]\n" +
'}\n'
}

Files being used:
```
<relevant yaml/js/csv go here>
```

**Comment by gabor386** [comment]
Hi! After setting up artillery on my machine and aws cli, getting some weird error about modules while running this script on aws lambda.

**Comment by at-vpd** [comment]
I am getting the same issue : 

    "worker error, id: 1 browserType.launch: Executable doesn't exist at /home/sbx_user1051/.cache/ms-playwright/chromium-1045/chrome-linux/chrome\n" +

**Comment by hassy** [comment]
@at-vpd Running Playwright is not supported on AWS Lambda yet. Issue to track is here: https://github.com/orgs/artilleryio/projects/2?pane=issue&itemId=7335761

**Comment by hassy** [comment]
fixed in #1846
